.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
the content of this file should follow this format: `vX.Y.Z`.
Example `v16.14.0`

Reply:
Fixed